redis/src/it/scala/monix/connect/redis/RedisIntegrationTest.scala

```diff
@@ -12,8 +12,7 @@ import org.scalatest.flatspec.AnyFlatSpec
 
 import scala.concurrent.duration._
 
-class RedisIntegrationTest
-  extends AnyFlatSpec with Matchers with BeforeAndAfterEach with BeforeAndAfterAll {
+class RedisIntegrationTest extends AnyFlatSpec with Matchers with BeforeAndAfterEach with BeforeAndAfterAll {
```

Comment:
I tried looking around but `hget` is the only one behaving like this. Will have a look again.

Reply:
Notice that sometimes code doc don't say it can return a null, but it actually does. 
For longs and booleans I believe they won't, but for `String` responses I think it is possible.

Reply:
Yeah, I checked all the functions listed in `RedisHashReactiveCommands` and if key doesn't exist then either they return `0` or return an emtpy `List` depending on the behavior.